src/status_im/chat/models/message.cljs

```diff
@@ -194,3 +194,46 @@
 (fx/defn send-messages
   [cofx messages]
   (protocol/send-chat-messages cofx messages))
+
+(defn flip-args [f]
+  (fn [x y] (f y x)))
+
+(defn message-ids->message-id-chat-id-map
```

Comment:
mhm , why is it so complex ? isn't it possible to have this in the format we want from status-go ?

Reply:
Currently, the backend only returns the `message-id` of the deleted messages. But messages are indexed by `chat-id` in the frontend model. Its not ideal. I'll check if I can re-write the backend. There is one more issue that needs to be solved.

Reply:
i mean its not urgent, i think thats fine